src/vrl/api/tests/test_selectielijst.py

```diff
@@ -54,3 +56,21 @@ def test_filter_procestype(self):
             response_data['results'][0]['procesType'],
             f'http://testserver{procestype_url}',
         )
+
+
+class ResultaatTests(APITestCase):
+
+    def test_get_resultaat(self):
+        """ test resultaat get api:
+            bewaartermijn is displayed correctly
+            procestermijn_opmerking is displayed correctly """
+        resultaat = ResultaatFactory.create(
+            bewaartermijn=relativedelta(years=10),
+            procestermijn_opmerking='5 of 10 jaar'
+        )
+        url = reverse('resultaat-detail', kwargs={'uuid': resultaat.uuid})
+        response = self.client.get(url)
```

Comment:
I recommend tests to be have a very clear pattern of setup - execution - assertions, with clear separation between each 'block'. So before and after the `response = self.client.get(url)` a blank line will ease the reading/interpreting of tests